contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -491,18 +574,18 @@ NXmpes(NXobject):
           varied axis may be for example energy, momentum, spatial coordinate, pump-probe
           delay, spin index, temperature, etc. The axes traces should be linked to the
           actual encoder position in NXinstrument or calibrated axes in NXprocess.
```

Comment:
Put definitions of common axis names here into the docstring.

Reply:
Done. Sorry for the confusion, I wanted to keep the existing definitions in the `NXmpes_data` base class, but forgot to add the common axis names into the docstring. Thanks @domna for the fix!